src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/BindToComponent_SpecifiesValueAndChangeEvent_WithoutMatchingProperties/TestComponent.codegen.cs

```diff
@@ -36,7 +36,7 @@ protected override void BuildRenderTree(Microsoft.AspNetCore.Components.RenderTr
 #line default
 #line hidden
             );
-            __o = Microsoft.AspNetCore.Components.BindMethods.SetValueHandler(__value => ParentValue = __value, ParentValue);
```

Comment:
There are no new tests here because this changes existing behaviour instead of adding new scenarios. `bind-...` to an element or a weak-typed-bind to a component always uses `EventCallback<>`